pkg/controllers/user/cis/clusterScanHandler.go

```diff
@@ -82,6 +89,27 @@ func (csh *cisScanHandler) Create(cs *v3.ClusterScan) (runtime.Object, error) {
 		v3.ClusterScanConditionFailed.Message(cs, "cannot run scan on a windows cluster")
 		return cs, nil
 	}
+
+	// Wait for complete pod removal from previous scan
+	removed := false
+	for retry := NumberOfRetriesForPodRemoval; retry > 0; retry-- {
+		pods, err := csh.podLister.List(
+			v3.DefaultNamespaceForCis,
+			labels.Set(SonobuoyMasterLabel).AsSelector(),
```

Comment:
q: does this mean we can only have master across all clusters?

Reply:
one master per cluster ... hence we make sure there is only one scan running in a cluster at any given time.